### ISSUE TITLE
Line thickness doesn't seem to work with GPU_Polygon

### ISSUE THREAD

**Issue by kirilledelman** [grimfang4/sdl-gpu#94]
Hi, thanks for a great library. Found a bug:

GPU_SetLineThickness( 4.0 );
GPU_Polygon( ... )

draws polygon with line thickness 1.0

**Comment by grimfang4** [comment]
Yeah...   Triangles and Polygons are the only shapes left that use GL_LINES.  Thanks for reporting this so it doesn't get forgotten!
